modules/power-monitoring-accessing-dashboards-developer.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-accessing-dashboards-developer_{context}"]
+= Accessing {PM-shortname} dashboards as a developer
+
+You can access {PM-shortname} dashboards from the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the {product-title} web console.
+* You have access to the cluster as a developer or as a user.
+* You have installed the {PM-operator}.
+* You have deployed {PM-kepler} in your cluster.
+* You have enabled monitoring for user-defined projects.
+* You have view permissions for the project that you are viewing the dashboard for.
+
+.Procedure
+
+. In the *Developer* perspective of the web console, go to *Observe* -> *Dashboards*.
+
+. From the *Dashboard* drop-down list, select the {PM-shortname} dashboard you want to see: 
+** *Power Monitoring / Overview*
+** *Power Monitoring / Namespace*
++
+[NOTE]
+====
+Only the namespaces that are accessible to developers in the {PM-shortname} dashboard are visible in the *Developer* perspective.
+====
```

Comment:
note seems to be indented for the bullet point. perhaps needs to be indented to the section itself?

Reply:
@vimalk78 I have removed the indentation from the bullet point.